UnitTests/PayPalTests/PaymentButton_Tests.swift

```diff
@@ -0,0 +1,25 @@
+import XCTest
+@testable import PayPal
+
+class PaymentButton_Tests: XCTestCase {
+
+    func testInit_whenPayPalButtonCreated_hasUIImageFromAssets() {
+        let payPalButton = PayPalButton()
+        XCTAssertEqual(payPalButton.imageView?.image, PaymentButtonImage.payPal.rawValue)
+    }
+
+    func testInit_whenPayPalCreditButtonCreated_hasUIImageFromAssets() {
+        let payPalCreditButton = PayPalCreditButton()
+        XCTAssertEqual(payPalCreditButton.imageView?.image, PaymentButtonImage.payPalCredit.rawValue)
+    }
+
+    func testInit_whenPayPalButtonCreated_hasUIColorFromAssets() {
+        let payPalButton = PayPalButton()
```

Comment:
I think we should try to follow our pattern of file-to-file unit test matching. So for example, we have a file `PayPalButton.swift` so it gets its own `PayPalButton_Tests.swift` and same for `PayPalCreditButton.swift` --> `PayPalCreditButton_Tests.swift`, for consistency.